### DISCUSSION TITLE
Accessibility for plots

### DISCUSSION THREAD

**Discussion by AndrewJConley** [NCAR/music-box-interactive-client#338] [Ideas]
In addition to ARIA for accessibility, we need some way for users to "read" (listen?) plots.  

Let's research what would work and be sustainable.

Some options so far:
1. https://accessiblegraphs.org/?lang=en
2. https://d3js.org/  with, say some of the techniques in https://fossheim.io/writing/posts/accessible-dataviz-d3-intro/

One of the first steps might be gaining more understanding of what we mean by accessibility for someone using our tool.  I.E., functional needs: https://www.w3.org/TR/wcag-3.0/#dfn-functional-need

**Comment by K20shores** [comment]
I recommend the d3 option

**Comment by K20shores** [comment]
#24 and #80 added d3. In addition, #109 turned us into a single page application. We need to check accessibility for the whole site and for our graphs.

https://observablehq.com/@9a849ed5a351d187/adventures-in-d3-accessibility
